src/types/patientsTypes.ts

```diff
@@ -3,4 +3,38 @@ export interface IPatient {
   name: string;
   lastName: string;
   phone: string;
+};
+
+export interface IPatientForm {
```

Comment:
Stop, I sow that in interface IPatientFormForBackend

Is it duplication?

Reply:
I delete it )

Reply:
it is not a duplicate
this is to change the types by some value